yarn-project/simulator/src/client/private_execution.test.ts

```diff
@@ -394,7 +400,20 @@ describe('Private Execution test suite', () => {
 
       const noteTypeId = StatefulTestContractArtifact.notes['ValueNote'].id;
 
-      const notes = [buildNote(60n, owner, storageSlot, noteTypeId), buildNote(80n, owner, storageSlot, noteTypeId)];
+      const notes = [
+        buildNote(
+          60n,
+          poseidon2Hash(ownerCompleteAddress.publicKeys.masterNullifierPublicKey.toFields()),
```

Comment:
i think it would be better to have 1 hash public key function which would get called everywhere. This could be tedious to update in all the tests if we decide to hash it differently.